Microsoft.Toolkit.Uwp/Helpers/SystemInformation.cs

```diff
@@ -180,7 +181,8 @@ public TimeSpan AppUptime
         /// Tracks information about the app's launch.
         /// </summary>
         /// <param name="args">Details about the launch request and process.</param>
-        public void TrackAppUse(LaunchActivatedEventArgs args)
+        /// <param name="xamlRoot">The XamlRoot object from your visual tree.</param>
+        public void TrackAppUse(IActivatedEventArgs args, XamlRoot xamlRoot = null)
```

Comment:
Since we use the activated args, should we make this method an extension to the Application class where they'd be calling this from anyway?

Reply:
I don't think that is a good idea. If a user want to use this outside of the context of the Application class, for example, within a Xaml Island app, then they can still provide the right information thru that interface. That is why I changed from the concrete class to the interface.